src/librustc_back/target/i686_apple_darwin.rs

```diff
@@ -12,7 +12,9 @@ use target::Target;
 
 pub fn target() -> Target {
     let mut base = super::apple_base::opts();
-    base.cpu = "yonah".to_string();
+    // Use i686 as default CPU. Clang uses the same default.
+    base.cpu = "i686".to_string();
```

Comment:
@ranma42 but if there can’t possibly be such a combination of darwin+x86 which uses anything pre-yonah, why bother targeting a decade-older CPU?